admin/src/js/controllers/targets.js

```diff
@@ -14,25 +13,6 @@ angular.module('controllers').controller('TargetsCtrl',
     $scope.loading = true;
     $scope.error = false;
 
-    const setEnabled = function(value) {
```

Comment:
In vain did I look for the existing tests associated with this functionality....  😭

Reply:
Heh, yeah... this is my favourite way to increase test coverage